docs/asset-store/docs/06-02-bucket.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Bucket
+type: Custom Resource
+---
+
+The `buckets.assetstore.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define the name of the cloud storage bucket for storing assets. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd buckets.assetstore.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource that defines the storage bucket configuration.
+
+```
+apiVersion: assetstore.kyma-project.io/v1alpha1
+kind: Bucket
+metadata:
+  name: my-bucket
+  namespace: default
+spec:
+  location: "us-east-1"
+  policy: `
+     {
+      "Version":"2012-10-17",
+      "Statement":[
+         {
+            "Action":[
+               "s3:GetObject"
+            ],
+            "Effect":"Allow",
+            "Principal":{
+               "AWS":[
+                  "*"
+               ]
+            },
+            "Resource":[
+               "arn:aws:s3:::my-bucket/*"
+            ],
+            "Sid":""
+         }
+      ]
+   }
+
+```
+
+## Custom resource parameters
+
+This table lists all the possible parameters of a given resource together with their descriptions:
+
+
+| Parameter   |      Mandatory      |  Description |
+|:----------:|:-------------:|:------|
+| **metadata.name** |    **YES**   | Specifies the name of the CR. |
+| **metadata.namespace** |    **YES**   | Specifies the Namespace in which the CR is available. |
+| **spec.location** |    **NO**   | xxx  Specifies the [location](https://docs.minio.io/docs/golang-client-api-reference.html#MakeBucket) under which the bucket is created. If the field is empty, the bucket is created under the default location. |
```

Comment:
Please replace the link with https://github.com/kyma-project/kyma/blob/master/components/assetstore-controller-manager/config/crds/assetstore_v1alpha1_bucket.yaml#L34